core/includes/form.inc

```diff
@@ -4748,8 +4753,18 @@ function theme_form_element($variables) {
   if (!isset($element['#title'])) {
     $element['#title_display'] = 'none';
   }
+
   $prefix = isset($element['#field_prefix']) ? '<span class="field-prefix">' . $element['#field_prefix'] . '</span> ' : '';
-  $suffix = isset($element['#field_suffix']) ? ' <span class="field-suffix">' . $element['#field_suffix'] . '</span>' : '';
+  if (isset($element['#field_suffix'])) {
+    // Set default class for all field suffixes.
+    $suffix_classes = array('field-suffix');
```

Comment:
Using this array, with the default `field-suffix` class ensures that we will not break existing suffixes, while at the same time we can conditionally specify additional, different classes for other suffixes we may want to target specifically in the future (and we can do the same for prefixes 😉 ).